jjb/anax/anax.yml

```diff
@@ -0,0 +1,21 @@
+- job:
+    name: jenkins-job
+    project-type: pipeline
+    defaults: global
+    node: 'ubuntu2204-builder-aws-1c-2g'
+    pipeline-scm:
+      script-path: "Jenkinsfile"
+      scm:
+        - git:
+            url: "https://github.com/open-horizon/anax.git"
+            clean: true
+    scm:
+      - git:
+          url: https://github.com/open-horizon/anax.git
+          branches:
+            - master
+          browser: auto
+          timeout: 20
+    triggers:
```

Comment:
Pipelines don't need a cron trigger; they use web hooks.

Reply:
Is it already there or I need to define the hook?

Reply:
It's created automatically. And if it's not, I'll have to create it manually in Github. But the Jenkins process is automatic.

Reply:
I think I need to keep the "triggers" to configure Jenkins build on push